src/main/kotlin/uk/gov/dluhc/printapi/rest/CertificateStatisticsController.kt

```diff
@@ -0,0 +1,44 @@
+package uk.gov.dluhc.printapi.rest
+
+import mu.KotlinLogging
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.CrossOrigin
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.dluhc.printapi.database.entity.SourceType
+import uk.gov.dluhc.printapi.mapper.CertificateStatisticsStatusMapper
+import uk.gov.dluhc.printapi.models.StatisticsResponse
+import uk.gov.dluhc.printapi.service.CertificateFinderService
+import uk.gov.dluhc.printapi.service.temporarycertificate.TemporaryCertificateFinderService
+
+private val logger = KotlinLogging.logger {}
+
+@RestController
+@CrossOrigin

```

Comment:
Not sure what the `@CrossOrigin`s do and if we need them for this controller.

Reply:
Probably not needed - we're not expecting requests to come from a browser to this endpoint. Although I'm not completely clear about how this interacts with the CORS config we've set at the API Gateway level.

Fine to leave.